src/sentry/static/sentry/app/views/performance/transactionSummary/transactionList.tsx

```diff
@@ -60,6 +72,30 @@ class TransactionList extends React.PureComponent<WrapperProps> {
     browserHistory.push(target);
   };
 
+  handleIncompatibleQuery = (incompatibleQuery: React.ReactNode) => {
+    const {organization} = this.props;
+    trackAnalyticsEvent({
+      eventKey: 'performance_views.summary.create_alert_incompatible',
+      eventName:
+        'Performance Views: Creating an alert from transaction summary was incompatible',
+      organization_id: parseInt(organization.id, 10),
+    });
+    this.setState({incompatibleQuery});
+  };
+
+  handleAlertClose = () => {
+    this.setState({incompatibleQuery: null});
+  };
+
+  handleCreateAlertSuccess = () => {
+    const {organization} = this.props;
+    trackAnalyticsEvent({
+      eventKey: 'performance_views.summary.create_alert',
+      eventName: 'Performance Views: Create Alert from Transaction Summary',
+      organization_id: parseInt(organization.id, 10),
```

Comment:
You actually don't need to parse the id. the backend for trackAnalyticsEvent will do this for you.